R/constants.R

```diff
@@ -489,7 +489,11 @@ gcamusa.DEFAULT_COEFFICIENT <- 1
 gcamusa.DEFAULT_MARKET <- "USA"
 gcamusa.GAS_ADJ_THRESH <- 5
 gcamusa.DIGITS_COST <- 4
+gcamusa.EFFICIENCY_PARTITION_YEAR <- 2005
 
+# Degree day norms (no idea where these came from)
+gcamusa.BASE_HDD_USA <- 4524
+gcamusa.BASE_CDD_USA <- 1215
```

Comment:
Anyone know about this? @pkyle @cahartin ?

Reply:
https://www.eia.gov/totalenergy/data/annual/showtext.php?t=ptb0107 (Normal, row, Total column)
https://www.eia.gov/totalenergy/data/annual/showtext.php?t=ptb0108

Reply:
👍 thanks pkyle